docker-machine-nfs.sh

```diff
@@ -239,7 +239,7 @@ checkMachineRunning ()
 {
   echoInfo "machine running ... \t\t\t"
 
-  machine_state=$(docker-machine ls $2 | sed 1d | grep "^$1\s" | awk '{print $4}')
+  machine_state=$(docker-machine ls -t $2 --filter "Name=$1" --format "{{.State}}")
```

Comment:
Will this search on the exact full name? It doesn't work as a wildcard? So if for example you have a machine called "test" and a machine "test1" ... will the filter only return 1 entry when filtering for "test" ?

Reply:
I'll check it now.

Reply:
Yes, you're right. I'll add `^` and `$` to fix it.

Reply:
I fixed it.